geometry_msgs/Point  Xs
float64[9]           covariance
uint32               id
float64              row_coord_px
float64              col_coord_px